src/clj/rems/migrations/convert_to_dynamic_applications.clj

```diff
@@ -0,0 +1,21 @@
+(ns rems.migrations.convert-to-dynamic-applications
+  (:require [clojure.java.jdbc :as jdbc]
+            [clojure.test :refer :all]
+            [conman.core :as conman]
+            [rems.db.core :as db :refer [*db*]]
+            [rems.db.workflow :as workflow]))
+
+(defn- use-db [jdbc-url f]
+  (binding [*db* (conman/connect! {:jdbc-url jdbc-url})]
+    (try
+      (f)
+      (finally
+        (conman/disconnect! *db*)))))
```

Comment:
hmm should this be somewhere else?

or should we just use mount as usual for the db connection?

Reply:
The `migrate-up` method is called by migratus. Currently `-main` does not `mount/start *db*` when running migrations, but that could be changed.

Best would be to use the same database connection as migratus uses. The Connection instance is inside the config map that `migrate-up` receives, but it would need to be wrapped inside a mock DataSource that always returns the same connection, because `*db*` expects a DataSource instead of Connection.

Reply:
This file will actually be removed. It's better to run the data migration as a separate command from the command line instead of as part of the schema migrations.